paddle/fluid/framework/operator.h

```diff
@@ -195,6 +199,10 @@ class ExecutionContext {
     return op_.Attr<T>(name);
   }
 
+  inline bool HasAttr(const std::string& name) const {
+    return op_.HasAttr(name);
+  }
```

Comment:
What's the purpose of adding `HasAttr` method? Why don't previous conv2d #8451 and pool #8879 PR need this method?

Reply:
Done. I removed all my changes. The idea was that it would be nice to have a function that would check existence of attribute in hashtable.